src/modules/collection/sagas.ts

```diff
@@ -100,7 +101,11 @@ function* handleSaveCollectionRequest(action: SaveCollectionRequestAction) {
     yield put(closeModal('CreateCollectionModal'))
     yield put(closeModal('EditCollectionNameModal'))
   } catch (error) {
-    yield put(saveCollectionFailure(collection, error.message))
+    const errorMessage = error.response.data.error || error.message
+    if (errorMessage === 'Name already in use') {
+      return yield put(saveCollectionFailure(collection, t('create_collection_modal.error_name_already_in_use')))
```

Comment:
This works, but it shouldn't be the sagas's responsibility to know how the message looks I think. I'd move this logic to the component that shows the error, having two calls to the same method with a translated version seems odd